tox.ini

```diff
@@ -195,25 +198,15 @@ commands = python -m aea.cli check-packages
 skipsdist = True
```

Comment:
above `hash-check` should be updated to use `open-aea[all]` and `open-aea-cli-ipfs` instead. Then also directly use  cli rather than via `python -m`